From b9dad08d29dc724527878b6749b1e67dafd47c9c Mon Sep 17 00:00:00 2001
From: Lionel Debieve <lionel.debieve@foss.st.com>
Date: Fri, 19 Nov 2021 16:42:52 +0100
Subject: [PATCH 0105/1695] plat-stm32mp1: do not generate stm32 files

Using the FIP with trusted firmware, the old mode using
stm32 files is deprecated so generation is no more required.
STM32 generation file is still possible using the
CFG_STM32MP15x_STM32IMAGE configuration flag.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 13ffacaa6ab26fcf4374df26a8e2d3059e83a621 plat-stm32mp1: do not generate stm32 files]
Signed-off-by: Lionel Debieve <lionel.debieve@foss.st.com>
Change-Id: Ibd25dcdc301b271a97938e609ea9949e299a54f1
---
 core/arch/arm/plat-stm32mp1/conf.mk | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/core/arch/arm/plat-stm32mp1/conf.mk b/core/arch/arm/plat-stm32mp1/conf.mk
index 7bcb38078..98b0207b3 100644
--- a/core/arch/arm/plat-stm32mp1/conf.mk
+++ b/core/arch/arm/plat-stm32mp1/conf.mk
@@ -383,3 +383,6 @@ CFG_STM32_DEBUG_ACCESS ?= $(CFG_TEE_CORE_DEBUG)
 ifeq ($(call cfg-all-enabled,CFG_STM32MP15 CFG_STM32MP13),y)
 $(error CFG_STM32MP13_CLK and CFG_STM32MP15_CLK are exclusive)
 endif
+
+# Generate the STM32 files
+CFG_STM32MP15x_STM32IMAGE ?= n
-- 
2.47.2

